### ISSUE TITLE
[Bug]: 

### ISSUE THREAD

**Issue by vladimirsim** [jasp-stats/jasp-issues#2090]
### JASP Version

0.17.1

### JASP Module

Descriptives

### What analysis are you seeing the problem on?

Descriptives, Machine Learning, etc.

### What OS are you seeing the problem on?

Windows 10

### Bug Description

When I load JASP (no matter if I run it as administrator or not) it starts normally. I load a database (no matter if from local drive or online) and it appears on screen just fine. For example, one of the databases which causes it to crash is the Telco customer churn in the Machine learning data library. If I only watch it or scroll it, it is fine. But when I press the 'Descriptives' button on the ribbon, or if I select analysis from the 'Machine learning' menu, then the program 'thinks' for 3 seconds and then it closes without any error message. I have not tried the other menus, but most probably it will be the same.

### Expected Behaviour

Should not close.

### Steps to Reproduce

1. Load a database. No matter local or from data library. JASP crashes will ALL databases which I have tried (2 in total). 
2. Open Descriptives or select a method from Machine learning. 
3. Don't do anything else. Just watch how JASP closes within 3 seconds. 
...

### Final Checklist

- [ ] I have included a screenshot showcasing the issue, if possible.
- [ ] I have included a JASP file (zipped) or data file that causes the crash/bug, if applicable.
- [X] I have accurately described the bug, and steps to reproduce it.

**Comment by vladimirsim** [comment]
JASP a second before it crashes due to the fact that I have opened the 'Descriptives' menu or the 'K-Nearest Neighbor regression' menu. The result is the same, no matter the data file or its location (network or local drive). I could not take a print screen from the computer, didn't work, so I took a photo with my phone.
![1681300661476](https://user-images.githubusercontent.com/130318189/231454047-22e9789a-f0b4-45ca-b6c0-d3553688b6c7.jpg)
![1681300661499](https://user-images.githubusercontent.com/130318189/231454056-301ace5a-1ec2-4c50-acff-d0b6043fb2f3.jpg)
